### PR TITLE
fix: replace sfdx with sf-release [semver:patch]

### PR DESCRIPTION
**Pull request by peternhale**

### CODE REVIEW

**Review by mshanemc — APPROVED** [review]

File: src/commands/halt-workflow-if-not-releasable.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  Command will set env var NEEDS_RELEASE accordingly
+
+steps:
+  - run:
+      name: Halt Workflow if not Releasable
+      command: |
+        NEEDS_RELEASE=$(sf-release npm:release:validate)
+        if [ "${NEEDS_RELEASE}" == "false" ]; then
+          echo "A Release is not necessary. Halting this workflow."
+          circleci-agent step halt
```
Comment (peternhale) [inline src/commands/halt-workflow-if-not-releasable.yml:18]: Simple is better

File: src/jobs/release-lerna-packages.yml [hunk 2]
```diff
@@ -77,6 +77,12 @@ steps:
         - ~/.cache/yarn
         - ~/.config/yarn
 
+  # - run npm:release:validate to see if a release is necessary - will set env var NEEDS_RELEASE
+  - when:
```
Comment (peternhale) [inline src/jobs/release-lerna-packages.yml:81]: Additional step to halt the workflow if not releasable

File: src/jobs/release-package.yml [hunk 3]
```diff
@@ -100,6 +100,12 @@ steps:
         - ~/.cache/yarn
         - ~/.config/yarn
 
+  # - run npm:release:validate to see if a release is necessary - will set env var NEEDS_RELEASE
+  - when:
```
Comment (peternhale) [inline src/jobs/release-package.yml:104]: Additional step to halt the workflow if not releasable

File: src/jobs/release-lerna-packages.yml [hunk 4]
```diff
@@ -77,6 +77,12 @@ steps:
         - ~/.cache/yarn
         - ~/.config/yarn
 
+  # - run npm:release:validate to see if a release is necessary - will set env var NEEDS_RELEASE
```
Comment (mshanemc) [inline src/jobs/release-lerna-packages.yml:80]: is the comment true that it still sets an env?

File: src/jobs/release-lerna-packages.yml [hunk 5]
```diff
@@ -144,13 +150,17 @@ steps:
   # when dryrun is false: run post-job-steps
   - when:
       condition:
-        not: <<parameters.dryrun>>
+        and:
+          - equal: [ true, $NEEDS_RELEASE ]
```
Comment (mshanemc) [inline src/jobs/release-lerna-packages.yml:154]: how does the env get persisted?

**Review by mshanemc — CHANGES_REQUESTED** [review]
I'm still confused on env persistence.

File: src/jobs/release-lerna-packages.yml (hunk 5 above)
Comment (peternhale) [reply src/jobs/release-lerna-packages.yml:154]: @mshanemc  Comment needed to be fixed to reflect reality. Thank you.

**Review by mshanemc — APPROVED** [review]

**Pull request merged**